src/components/requests/map.tour.vue

```diff
@@ -0,0 +1,92 @@
+<template>
+  <div ref="MapTourRequest">
+    <div>Move the map along route for visualization purposes</div>
+    <button ref="btnMapTourRequest" class="btn btn-primary exampleready" @click="mapTourRequest">MapTourRequest</button>
+    <div>
+      <a id="mapTour" href="" @click="getLink">To the documentation</a>
+    </div>
+    <code-component snippet="
+      const x = 488704;
+      const y = 6939136;
+      const params = [
+        [
+          {
+            lon: x,
+            lat: y,
+            duration: 5000
+          },
+          {
+            lon: x - 150000,
+            lat: y - 150000
+          },
+          {
+            lon: x + 75000,
+            lat: y,
+            animation: 'zoomPan',
+            zoom: 6
+          }
+        ],
+        {
+          zoom: 8,
+          animation: 'fly',
+          duration: 3000,
+          cancelled: () => console.log('cancelled'),
+          completed: () => console.log('completed'),

```

Comment:
Functions can't be sent through RPC so these can't work. I think there was a MapTourEvent or something like that which lets you know about completion.